ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,898 @@
+            * @type      {string}
+            * @since     next
+            */
+            format: '{value}',
+
+            /**
+             * Callback function to format the breadcrumb text from scratch.
+             *
+             * @param {Highcharts.Breadcrumbs} breadcrumbs
+             *        The breadcrumbs instance
+             *
+             * @return {string}
+             *         Formatted text or false
+             * @since    next
+             */
+
+            /**
+            * Positioning for the button row. The breadcrumbs buttons
+            * will be aligned properly for the default chart layout
+            * (title,  subtitle, legend, range selector) for the custom chart
+            * layout set the position properties.
+            */
+            position: {
+                /**
+                 * Horizontal alignment of the breadcrumbs buttons.
+                 *
+                 * @type {Highcharts.AlignValue}
+                 */
+                align: 'left',
+
+                /**
+                 * Vertical alignment of the breadcrumbs buttons.
+                 *
+                 * @type {Highcharts.VerticalAlignValue}
+                 */
+                verticalAlign: 'top',
+
+                /**
+                 * The X offset of the breadcrumbs button group.
+                 */
+                x: 0,
+
+                /**
+                 * The Y offset of the breadcrumbs button group.
+                 */
+                y: 0
+            },
+
+            /**
+            * A CSS styles for all breadcrumbs.
+            *
+            * @type      {string}
+            * @since     next
+            */
+            style: {
+                'color': '#666666',
+                'position': 'absolute'
+            },
+
+            /**
+             * A predefined shape or symbol for the separator. Other possible
+             *  values are `'circle'`, `'square'`,`'diamond'`.
+             * Custom callbacks for symbol path generation can also be added to
+             * `Highcharts.SVGRenderer.prototype.symbols`. The callback is then
+             * used by its method name.
+             *
+             * Additionally, the URL to a graphic can be given on this form:
+             * `'url(graphic.png)'`. Note that for the image to be applied to
+             * exported charts, its URL needs to be accessible by the export
+             * server. Recommended size of image is 10x10px.
+             *
+             * @type      {string}
+             * @since     next
+             */
+            separator: 'triangle',
+
+            /**
+            * A useHTML property.
+            *
+            * @type      {boolean}
+            * @since     next
+            */
+            useHTML: false,
+
+            /**
+            * The zIndex of the group.
+            *
+            * @type      {boolean}
+            * @since     next
+            */
+            zIndex: 7
```

Comment:
Perhaps `8` would be better (to be higher than the selection marker in zoom). And we might as well increase tooltips label from 8 to 9 (so tooltip will be above the breadcrumbs).